src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Builder/OpenTelemetryBuilderOtlpExporterExtensions.cs

```diff
@@ -56,7 +58,9 @@ public static IOpenTelemetryBuilder UseOtlpExporter(
     {
         Guard.ThrowIfNull(baseUrl);
 
+#pragma warning disable CA1062 // Validate arguments of public methods
         return UseOtlpExporter(builder, name: null, configuration: null, configure: otlpBuilder =>
+#pragma warning restore CA1062 // Validate arguments of public methods
```

Comment:
@rajkumar-rangaraj consider a follow up item to validate these arguments.

Reply:
I think that it is validated in the internal method. Only netstandard2.1 target is not working with such detection.